plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -181,3 +164,20 @@ export default compose(
 		};
 	} )
 )( Connect );
+
+export const prefetchJetpackAuthData = ( select, redirectUrl ) => {
+	const { getJetpackAuthUrl, isResolving, getOnboardingError } = select(
+		ONBOARDING_STORE_NAME
+	);
+
+	const queryArgs = {
+		redirectUrl: redirectUrl || window.location.href,
+		from: 'woocommerce-services',
+	};
+
+	return {
+		error: getOnboardingError( 'getJetpackAuthUrl' ) || '',
+		isRequesting: isResolving( 'getJetpackAuthUrl', [ queryArgs ] ),
```

Comment:
The `getJetpackAuthUrl` function from the Onboarding data store does not use a `setIsRequesting` call like the function from the Plugins data store used to so we use the `@wordpress/data`-generic `isResolving` instead.